styles/globals.css

```diff
@@ -3,12 +3,28 @@
   src: local("AlmondCream"), url(../public/AlmondCream.woff) format("woff");
 }
 
+@font-face {
+  font-family: "Gabriola";
+  src: local("Gabriola"), url(../public/Gabriola.woff) format("woff");
+}
+
+:root {
+  --mainbackground-color: #50c4b7;
+  --secondary-color: #fff;
+  --link-color: #674b14;
+  --font-color: #000;
+  --quizbutton-color: #2537c7;
+  --quizbutton-false-color: #f80606;
+  --quizbutton-true-color: #699716;
+}
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue,
+    "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```

Comment:
`Franklin Gothic Medium` and `Arial Narrow` are not imported.
And it is very unlikely that it is used because of this long list of font families.